.vscode/launch.json

```diff
@@ -26,7 +26,7 @@
             "runtimeExecutable": "${execPath}",
             "args": [
                 "--extensionDevelopmentPath=${workspaceFolder}",
-                "--extensionDevelopmentPath=${workspaceFolder}/../vscode-azureresourcegroups"
```

Comment:
You shouldn't remove `vscode-azureresourcegroups`. The host always implies the Resources extension.

If anything, you should just make a new launch task that includes container apps for testing the API. Launch Extension + API or something like that.